development/tools/jobs/testinfra/validate_test.go

```diff
@@ -10,6 +10,7 @@ import (
 
 func TestValidateProwPresubmit(t *testing.T) {
 	// WHEN
+	t.Skip()
```

Comment:
Are we temporarily skipping this or why do we not remove it completely in this case?

Reply:
New tests will be added instead of this one. Its temporarily disabled